### PR TITLE
Some Requested Style Changes.

### PR DESCRIPTION
**Pull request by bd82**

### CODE REVIEW

File: lib/dsl/validator.js [hunk 1]
```diff
@@ -256,7 +259,7 @@ class JDLSyntaxValidatorVisitor extends BaseJDLCSTVisitorWithDefaults {
   checkConfigPropSyntax(key, value) {
     const propertyName = key.tokenType.tokenName;
     const validation = configPropsValidations[propertyName];
-    if (_.isUndefined(validation)) {
+    if (validation === undefined) {
```
Comment (bd82) [inline lib/dsl/validator.js]: I meant to check for undefined here, not null, empty strings, zero ...

Comment (MathieuAA) [reply lib/dsl/validator.js]: `!validation`

Comment (MathieuAA) [reply lib/dsl/validator.js]: It depends on what `configPropsValidations` returns. It's a custom function, so we know and can control what it returns.

Comment (bd82) [reply lib/dsl/validator.js]: fixed

**Pull request merged**